core/src/main/web/app/helpers/core.js

```diff
@@ -708,26 +719,36 @@
         if (!msgHeader) {
           msgHeader = "Question...";
         }
-        var close = function(result) {
-          if (result === "OK") {
-            okCB ? okCB() : null;
-          } else { // cancel
-            cancelCB ? cancelCB() : null;
+        var callback = function(result) {
+          if (((result == "OK") || (result == 0)) && okCB) {
```

Comment:
instead of changing this check to handle two values for result can we just normalize the value before it gets into this block of logic?